README.md

```diff
@@ -1096,9 +1098,7 @@ In the example below, the checkbox and submit button have been wrapped in a
 <%= bootstrap_form_for(@user, layout: :horizontal, label_col: "col-sm-2", control_col: "col-sm-10") do |f| %>
   <%= f.email_field :email %>
   <%= f.password_field :password %>
-  <%= f.form_group do %>
-    <%= f.check_box :remember_me %>
-  <% end %>
+  <%= f.check_box :remember_me %>
```

Comment:
The text above says that the check box is wrapped in a form group for alignment, when this is apparently no longer necessary. I think the new code is fine but the text should be fixed.

Reply:
Fixed the text now.